src/Okta.Sdk.IntegrationTests/TrustedOriginScenarios.cs

```diff
@@ -0,0 +1,241 @@
+            trustedOrigins.Count.Should().BeGreaterThan(0);
+        }
+
+        [Fact]
+        public async Task CreateOrigin()
+        {
+            var client = TestClient.Create();
+            var testTrustedOriginName = $"{dotnetSdkPrefix}_{nameof(CreateOrigin)}_Test";
+            var createdTrustedOrigin = await client.TrustedOrigins.CreateOriginAsync(
+                new TrustedOrigin
+                {
+                    Name = testTrustedOriginName,
+                    Origin = "http://example.com",
+                    Scopes = new List<IScope>()
+                    {
+                        new Scope()
+                        {
+                            Type = "CORS",
+                        },
+                        new Scope()
+                        {
+                            Type = "REDIRECT",
+                        },
+                    },
+                });
+
+            try
+            {
+                createdTrustedOrigin.Should().NotBeNull();
+                createdTrustedOrigin.Name.Should().Be(testTrustedOriginName);
+                createdTrustedOrigin.Scopes.Count.Should().Be(2);
+                createdTrustedOrigin.Scopes.FirstOrDefault(scope => scope.Type == ScopeType.Cors).Should().NotBeNull();
+                createdTrustedOrigin.Scopes.FirstOrDefault(scope => scope.Type == ScopeType.Redirect).Should().NotBeNull();
+            }
+            finally
+            {
+                await client.TrustedOrigins.DeleteOriginAsync(createdTrustedOrigin.Id);
+            }
+        }
+
+        [Fact]
+        public async Task DeleteOrigin()
+        {
+            var client = TestClient.Create();
+            var testTrustedOriginName = $"{dotnetSdkPrefix}_{nameof(DeleteOrigin)}_Test";
+            var createdTrustedOrigin = await client.TrustedOrigins.CreateOriginAsync(
+                new TrustedOrigin
+                {
+                    Name = testTrustedOriginName,
+                    Origin = "http://example.com",
+                    Scopes = new List<IScope>()
+                    {
+                        new Scope()
+                        {
+                            Type = "CORS",
+                        },
+                        new Scope()
+                        {
+                            Type = "REDIRECT",
+                        },
+                    },
+                });
+
+            var retrievedTrustedOrigin = await client.TrustedOrigins.GetOriginAsync(createdTrustedOrigin.Id);
+            retrievedTrustedOrigin.Should().NotBeNull();
+            await client.TrustedOrigins.DeleteOriginAsync(createdTrustedOrigin.Id);
+
+            var ex = await Assert.ThrowsAsync<OktaApiException>(() => client.TrustedOrigins.GetOriginAsync(createdTrustedOrigin.Id));
+            ex.StatusCode.Should().Be(404);
+        }
+
+        [Fact]
+        public async Task GetOrigin()
+        {
+            var client = TestClient.Create();
```

Comment:
To test this scenario, let's create a Trusted Origin and verify if it is retrieved by `GetOriginAsync`. We shouldn't rely on existing origins.